aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -104,6 +104,26 @@ const connection = new signalR.HubConnectionBuilder()
 > [!NOTE]
 > At this time, there are no configuration options for the MessagePack protocol on the JavaScript client.
 
+### Java client
```

Comment:
Does anyone know why this doc repeats itself 3 times?

Reply:
Yeah, different versions. Each one is between a `::: moniker range=">= aspnetcore-5.0"` and `::: moniker-end` and has different content based on which version you're referencing

Reply:
Ah, didn't notice that. I'll remove the new stuff from the pre-5.0 sections